docs/README.md

```diff
@@ -69,9 +69,9 @@ Copy the example stated below and paste it into your theme's desired template, m
 }
 ```
 
-:warning: *Notice that the Product Specifications' blocks necessarily need a Product context in order to work properly since they handle product data. Therefore, when declaring them, be sure that they are in a theme template in which this context is available, such as the `store.product`.*
+> ⚠️ _Note that the Product Specification blocks need a Product context to work properly because they handle product information. Therefore, when declaring them, be sure that they are in a theme template in which this context is available, such as `store.product`._
```

Comment:
```suggestion
>⚠️ The Product Specification blocks need a Product context to work properly because they handle product information. Therefore, when declaring them, be sure that they are in a theme template in which this context is available, such as `store.product`.
```